File: backend/src/main/java/com/example/shoestore/dto/response/OrderDetailResponse.java
package com.example.shoestore.dto.response;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderDetailResponse {
    private Integer id;
    private BigDecimal subtotal;
    private BigDecimal discountAmount;
    private BigDecimal shippingFee;
    private BigDecimal finalAmount;
    private String shippingName;
    private String shippingPhone;
    private String shippingAddress;
    private String status;
    private LocalDateTime orderDate;
    private List<OrderItemDetailResponse> items;
}

File: android/app/src/main/java/com/example/shoeshop/dto/respone/OrderDetailResponse.kt
package com.example.shoeshop.dto.respone

data class OrderDetailResponse(
    val id: Int,
    val subtotal: Double,
    val discountAmount : Double,
    val shippingFee: Double,
    val finalAmount: Double,
    val shippingName: String,
    val shippingPhone: String,
    val shippingAddress: String,
    val status: String,
    val orderDate: String,
    val items: List<OrderItemDetailResponse>
)